### ISSUE TITLE
Random values made test error.

### ISSUE THREAD

**Issue by keisuke-umezawa** [pfnet/chainer#2383]
As mentiond in PR #2374, 

There is the same issue in [test_fmod](https://github.com/pfnet/chainer/blob/master/tests/chainer_tests/functions_tests/math_tests/test_fmod.py) with #2381. 
I only changed the documentation codes, but one of the python version (Linux Python 3.4) faild.

This is the error codes of https://travis-ci.org/pfnet/chainer/jobs/208596535 .
```
======================================================================
FAIL: test_backward_cpu (test_fmod.TestFmod_param_0)  parameter: {'shape': (3, 2), 'dtype': <class 'numpy.float16'>}
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.4.2/lib/python3.4/site-packages/chainer/testing/condition.py", line 71, in wrapper
    fail()
  File "/home/travis/virtualenv/python3.4.2/lib/python3.4/site-packages/chainer/testing/condition.py", line 51, in fail
    instance.fail(msg)
nose.proxy.AssertionError: 
Fail: 3, Success: 0
The first error message:
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.4.2/lib/python3.4/site-packages/chainer/testing/condition.py", line 57, in <lambda>
    lambda: f(*args, **kwargs),
  File "/home/travis/build/pfnet/chainer/tests/chainer_tests/functions_tests/math_tests/test_fmod.py", line 83, in test_backward_cpu
    self.check_backward_cpu(F.fmod)
  File "/home/travis/build/pfnet/chainer/tests/chainer_tests/functions_tests/math_tests/test_fmod.py", line 44, in check_backward_cpu
    self.check_backward(op, self.x, self.divisor, self.gy)
  File "/home/travis/build/pfnet/chainer/tests/chainer_tests/functions_tests/math_tests/test_fmod.py", line 41, in check_backward
    rtol=5e-3, dtype=numpy.float64)
  File "/home/travis/virtualenv/python3.4.2/lib/python3.4/site-packages/chainer/gradient_check.py", line 265, in check_backward
    testing.assert_allclose(gx, x.grad, atol=atol, rtol=rtol)
  File "/home/travis/virtualenv/python3.4.2/lib/python3.4/site-packages/chainer/testing/array.py", line 24, in assert_allclose
    x, y, atol=atol, rtol=rtol, verbose=verbose)
  File "/home/travis/virtualenv/python3.4.2/lib/python3.4/site-packages/numpy/testing/utils.py", line 1297, in assert_allclose
    verbose=verbose, header=header)
  File "/home/travis/virtualenv/python3.4.2/lib/python3.4/site-packages/numpy/testing/utils.py", line 665, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Not equal to tolerance rtol=0.005, atol=0.0005
(mismatch 100.0%)
 x: array([[ 0.349365, -0.184937],
       [ 0.4021  ,  0.      ],
       [ 1.289542, -0.974609]])
 y: array([[ 0.349365, -0.184937],
       [ 0.4021  ,  0.      ],
       [ 4.675781, -0.974609]], dtype=float16)
```

**Comment by keisuke-umezawa** [comment]
I think the grad values are too small. https://github.com/pfnet/chainer/blob/master/tests/chainer_tests/functions_tests/math_tests/test_fmod.py#L69

**Comment by keisuke-umezawa** [comment]
It will be resolved by #2402. I close this issue.
